### PR TITLE
Add ns-live

### PR DESCRIPTION
**Pull request by dbenmore**
https://github.com/ConnectedHomes/nucleus/issues/2242

### CODE REVIEW

File: docs/components/ns-live.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+description: Organism | Live component.
+---
+
+import { StorybookStory } from '../../includes/storybook-story.js'
+import { Tokens } from '../../includes/tokens.js'
+import { ComponentPlacement } from '../../includes/component-placement.js'
+
+## Introduction
+
+> ns-live enables dynamic updating of content.
```
Comment (PtitBen) [inline docs/components/ns-live.md:11]: Can we have something a little more descriptive of what it allows?

> `ns-live` is a component that enables you to show a busy message state when dynamically updating content if your expect that action to take some time, or if you want to grab a users' attention on it.

(there is probably a better way to word it 😄 )

File: docs/components/ns-live.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Accessibility
+
+This component adds the `aria-live` attribute to itself. Screen readers will announce all changes that occur within this component
+
+## Content guidance
```
Comment (PtitBen) [inline docs/components/ns-live.md:17]: Should we have something about the loading animated icon?

File: docs/components/ns-live.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+| Key | Field type | Guidelines |
+| :--- | :--- | :--- |
+| A | Message | Keep this under 10 words and describe to the user what is happening e.g. Updating your quote |
+
+## Best practice
+
+| 💚 Do's | 💔 Don'ts |
+| :---  | :---  |
+| Use to wrap small pieces of dynamic content | Wrap large parts of the UI  |
+| Wrap inline elements (`<p>, <h1-6>`)| Place inside inline elements (`<p>, <h1-6>`) |
```
Comment (andij) [inline docs/components/ns-live.md:28]: `<p>` and `<h-6>` are block elements.

File: docs/components/ns-live.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+| Use as a slot | Use for media |
+| Keep the contents of ns-live simple |  |
```
Comment (andij) [inline docs/components/ns-live.md:30]: Include backticks around component name.
```suggestion
| Keep the contents of `ns-live` simple |  |
```

Comment (dbenmore) [reply docs/components/ns-live.md]: Done

File: docs/components/ns-live.md (hunk 3 above)
Comment (dbenmore) [reply docs/components/ns-live.md]: Reworded

File: docs/components/ns-live.md (hunk 2 above)
Comment (dbenmore) [reply docs/components/ns-live.md:17]: Added

File: docs/components/ns-live.md (hunk 1 above)
Comment (dbenmore) [reply docs/components/ns-live.md]: Reworded to make it a bit more descriptive - let me know if it's ok?

Comment (PtitBen) [reply docs/components/ns-live.md]: All good ! :) Thanks dude!

**Review by PtitBen — APPROVED** [review]

**Comment by RobTobias123** [comment]
Image of busy state for content guidance has been added.

**Pull request merged**